src/index.js

```diff
@@ -78,4 +78,13 @@ const resolveParams = args => {
   return cachePlugin(args)
 }
 
+// expose plugin as default
 export default resolveParams
+
+// expose action enhancer
+export function cacheAction(action) {
```

Comment:
An arrow function would improve readability.

What do you think, @superwf ?

```js
const cacheAction = (action) => (context, payload) => { ... };
```

Reply:
Totally agreed